runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeResponse.kt

```diff
@@ -0,0 +1,34 @@
+package io.openruntimes.kotlin
+
+import io.javalin.plugin.json.JsonMapper
+
+class RuntimeResponse(
+    var data: String = "",
+    var statusCode: Int = 200
+) {
+    fun json(data: Map<String, Any?>, statusCode: Int = 200): RuntimeResponse {
+        this.data = mapper?.toJsonString(data) ?: ""
```

Comment:
This is because we can not have the `RuntimeResponse.data` be a `String` _or_ `Map` with static typing, so we opt for the common format and serialise the data to string.

Runtimes that return JSON objects would likely just be serialising in an internal method